runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/work/processing/StreamingWorkScheduler.java

```diff
@@ -300,6 +301,10 @@ private void processWork(ComputationState computationState, Work work) {
     }
   }
 
+  private void processWork(ComputationState computationState, Work work) {
+    processWork(computationState, work, ImmutableList.of());
```

Comment:
instead of this delegating to the one with latencies how about that one delegates to this?

Reply:
done